graph_spec.rb

```diff
@@ -52,5 +52,23 @@
 
       expect(g.topological_sort).to eq(expected)
     end
+    it 'raises an UnsortableCyclicGraphError when the graph has cycles' do
```

Comment:
The rest of this file is using double quotes, so I think we should use those here, too.

I also like a line of whitespace separating my `it` blocks.